cob_navigation_config/cob3-2/costmap_common_params.yaml

```diff
@@ -1,30 +1,97 @@
+    data_type: LaserScan                # The data type associated with the topic, default: "PointCloud"
+    marking: true                       # Whether or not this observation is used to mark obstacles, default: true
+    clearing: true                      # Whether or not this observation is used to clear out freespace, default: false
+
+  # Parameters of the source: cloud_cam3d
+  cloud_cam3d:
+    sensor_frame: head_cam3d_link       # The frame of the origin of the sensor, default: ""
+    topic: /voxel_grid/output           # The topic on which sensor data comes in for this source, default: source_name
+    data_type: PointCloud2              # The data type associated with the topic, default: "PointCloud"
+    marking: true                       # Whether or not this observation is used to mark obstacles, default: true
+    clearing: true                      # Whether or not this observation is used to clear out freespace, default: false
+    min_obstacle_height: 0.12           # Minimum height in meters of a sensor reading considered valid, default: 0.0
+
+  # The default maximum distance from the robot at which an obstacle will be inserted into the cost map in meters
+  obstacle_range: 5.0 # default: 2.5
+
+  # The default range in meters at which to raytrace out obstacles from the map using sensor data
+  raytrace_range: 5.5 # default: 3.0
+
+  # What map type to use. "voxel" or "costmap" are the supported types, with the difference between them being a 3D-view of the world vs. a 2D-view of the world
+  map_type: voxel
+
+  publish_voxel_map: true
+
+  # The following parameters are only required if you have a VoxelLayer configured as ObstacleLayer
+  # Voxel Parameter: The z origin of the map in meters
+  origin_z: 0.0 # default: 0.0
+
+  # Voxel Parameter: The z resolution of the map in meters/cell
+  z_resolution: 0.23 # default: 0.2
+
+  # Voxel Parameter: The number of voxels to in each vertical column, the height of the grid is z_resolution * z_voxels
+  z_voxels: 2 # default: 10
+
+  # Voxel Parameter: The number of unknown cells allowed in a column considered to be "known"
+  unknown_threshold: 2 # default: ~<name>/z_voxels
 
-laser_scan_front: {sensor_frame: base_laser_front_link, topic: /base_laser_front/scan, data_type: LaserScan, marking: true, clearing: true}
+  # Voxel Parameter: The maximum number of marked cells allowed in a column considered to be "free"
+  mark_threshold: 0 # default: 0
 
-laser_scan_rear: {sensor_frame: base_laser_rear_link, topic: /base_laser_rear/scan, data_type: LaserScan, marking: true, clearing: true}
+#
+# Configure the inflation_layer
+#
+inflation_layer:
 
-laser_scan_top: {sensor_frame: base_laser_top_link, topic: /base_laser_top/scan, data_type: LaserScan, marking: true, clearing: true}
+  # The radius in meters to which the map inflates obstacle cost values.
+  inflation_radius: 0.55 # default: 0.55
```

Comment:
I know that value for inflation_radius is default, but makes no sense with the robot's circumscribed radius already larger than 0.55m. It should be inscribed_radius <= circumscribed_radius <= inflation_radius 
Also cost_scaling_factor is not properly chosen, a value of 8.01 rather suggests an inflation_radius of about 1.0m.

@ipa-mig as those are default values, I'd suggest to keep the values that way for this PR. But we should discuss this, and make a separate PR for updating these values when indicated.